### ISSUE TITLE
Build list of Y coords in advance

### ISSUE THREAD

**Issue by simoncozens** [simoncozens/atokern#1]
And cache it. Massive speed up.

**Comment by davelab6** [comment]
Can this be closed?

**Comment by simoncozens** [comment]
Nope, still needs to happen. Affected function is `glyph_to_sb` in `sidebearings.py`. Notice how it builds a massive list and then throws most of it away, because it's easier than thinking about the fencepost problem.
